package.json

```diff
@@ -76,6 +76,7 @@
     "gulp": "^4.0.2",
     "gulp-babel": "^8.0.0",
     "gulp-eslint": "^5.0.0",
+    "gulp-if": "^3.0.0",
```

Comment:
It looks like this dependency was added, but `package-lock.json` hasn't been modified. Could you rebuild the lockfile using `npm install` and commit the lockfile changes?

Reply:
Can you also update the `package-lock.json`?